src/app/expand-collapse-all/expand-collapse-all.component.ts

```diff
@@ -45,4 +45,8 @@ export class ExpandCollapseAllComponent implements OnInit {
     this.ccdService.updateShowDOMSource(this.showDOMSource);
   }
 
+  exportLinks() {
```

Comment:
I'm not sure exportLinks is appropriate in "expand-collapse-all" component. What do you think of changing this into options component or something similar?

Reply:
Taking care of this in a larger refactor branch